### PR TITLE
Adds support for NOMKSTREAM and MINID options in xadd.

### PR DESCRIPTION
**Pull request by sgarciac**
The 'LIMIT' option would be still missing but this PR takes xadd closer to completeness.

### CODE REVIEW

File: redis/src/streams.rs [hunk 1]
```diff
@@ -34,6 +34,59 @@ impl ToRedisArgs for StreamMaxlen {
...
 }
 
+/// Builder options for [`xadd_options`] command.
+///
+/// [`xadd_options`]: ../trait.Commands.html#method.xadd_options
+///
+#[derive(Default, Debug)]
+pub struct StreamAddOptions {
+    /// Set MAXLEN <threshold> cmd arg.
+    maxlen: Option<StreamMaxlen>,
```
Comment (djc) [inline redis/src/streams.rs:44]: I think we should use snake_case names (`max_len`, `min_id`, `no_mk_stream`) for both the fields and the methods here.

**Review by djc — COMMENTED** [review]
Looks pretty good to me, modulo some naming nits. Thanks!

**Comment by jaymell** [comment]
Can we add some tests?

**Comment by sgarciac** [comment]
ok :+1:

**Comment by nihohit** [comment]
@sgarciac can you rebase and add tests?

**Comment by nihohit** [comment]
fixed in https://github.com/redis-rs/redis-rs/pull/1361

**Pull request closed without merging**